README.md

```diff
@@ -0,0 +1,230 @@
+- [Support](#support)
+- [License](#license)
+
+## Features
+
+- **Decentralized Kubernetes Management**: Deploy and manage Kubernetes clusters on TFGrid
+- **Multi-Component Architecture**: Backend (Go), Frontend (Vue.js), CRDs, and networking plugins
+- **Monitoring & Observability**: Integrated Prometheus and Grafana for metrics and dashboards
+- **Custom Networking**: Mycelium CNI plugin for peer-to-peer networking
+- **Ingress Management**: Custom ingress controller for traffic routing
+- **RESTful APIs**: Comprehensive backend APIs for cluster operations
+- **Web UI**: Modern Vue.js frontend for cluster management
+- **Docker Integration**: Containerized deployment with Docker Compose
+- **Configuration Management**: Flexible configuration via files, environment variables, and CLI flags
+
+## Architecture Overview
+
+```text
++-------------------+    +-------------------+    +-------------------+
+|   Frontend UI     |    |   Backend APIs    |    |      TFGrid       |
+|    (Vue.js)       |--->|      (Go)         |<-->|  (Decentralized)  |
++-------------------+    +-------------------+    +-------------------+
+         |                      |                        |
+         |                      |                        |
+         |                 +----v----+                   |
+         |                 | Metrics |                   |
+         |                 | /metrics|                   |
+         |                 +----+----+                   |
+         |                      |                        |
+         v                      v                        v
++-------------------+    +-------------------+    +-------------------+
+|   Monitoring      |    |   CRDs &          |    |   Networking      |
+| (Prometheus/Grafana)|    | Controllers      |    |   (Mycelium)      |
++-------------------+    +-------------------+    +-------------------+
+```
+
+## Prerequisites
+
+Before getting started, ensure you have the following installed:
+
+- **Go**: Version 1.19 or later ([Download](https://golang.org/dl/))
+- **Node.js**: Version 18 or later ([Download](https://nodejs.org/))
+- **Docker**: Latest stable version ([Download](https://docker.com/))
+- **Docker Compose**: Latest version ([Install](https://docs.docker.com/compose/install/))
+- **Git**: For cloning the repository
+
+## Getting Started
+
+### 1. Clone the Repository
+
+```bash
+git clone https://github.com/codescalers/kubecloud.git
+cd kubecloud
+```
+
+### 2. Backend Setup
+
+Navigate to the backend directory and follow the setup instructions:
+
+```bash
+cd backend
+# Copy example configuration
+cp config-example.json config.json
+# Edit config.json with your settings (see backend/README.md for details)
+# Build and run
+make build
+./main
+```
+
+The backend will start on the configured port (default: 8080).
+
+### 3. Frontend Setup
+
+Navigate to the frontend directory and install dependencies:
+
+```bash
+cd ../frontend/kubecloud
```

Comment:
```suggestion
cd ./frontend/kubecloud
```

Reply:
According to the previous step, this is right.